test/parsing/errors/invalidNumericLiteral-brief.good

```diff
@@ -0,0 +1 @@
+invalidNumericLiteral.chpl:1: syntax error: error building numeric literal: integer literal overflow: '100000000000000000000000000000' is too big for type 'uint64'
```

Comment:
Notably, this reports `uint64` which is not the Chapel syntax.

Reply:
Good catch. I see this is part of the error message that was previously being reported, generated [here](https://github.com/chapel-lang/chapel/blob/edbd53ff8aab3096c2fc6d8b0593c7676a21b357/frontend/lib/types/Param.cpp#L493) (as well as at other points in that file).

I get why we might not want this to appear in a user-facing error message, but simultaneously it would feel inappropriate to report a Chapel type here since this error is generated if `sscanf` is unable to ingest the literal. We could cut the type off altogether and just say `integer literal overflow: '10000.....' is too big`, which feels a little bit too simple for me but perhaps more appropriate. Let me know what you think about the idea of changing to that in this PR.

Reply:
Or, we could try spell it out in words "64-bit signed integer" or something.

Reply:
I like the spelling it out route, going to go for that (it looks like it's unsigned however).